web/src/main/java/edu/softserve/zoo/exception/SecurityReason.java

```diff
@@ -0,0 +1,21 @@
+package edu.softserve.zoo.exception;
+
+import edu.softserve.zoo.exceptions.ExceptionReason;
+
+/**
+ * @author Ilya Doroshenko
+ */
+public enum SecurityReason implements ExceptionReason {
+    UNAUTHORIZED("You don't have rights to perform this request.");
```

Comment:
Should i somehow map Spring Security's native exceptions to ExceptionReasons and return to user message specified here, instead of built-in security exception message?